starter-core/src/main/java/io/micronaut/starter/feature/logging/Slf4jJulBridge.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2017-2023 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.starter.feature.logging;
+
+import io.micronaut.core.annotation.NonNull;
+import io.micronaut.starter.application.ApplicationType;
+import io.micronaut.starter.application.generator.GeneratorContext;
+import io.micronaut.starter.build.dependencies.Dependency;
+import io.micronaut.starter.feature.logging.template.julToSlf4j;
+import io.micronaut.starter.template.RockerTemplate;
+import jakarta.inject.Singleton;
+
+@Singleton
+public class Slf4jJulBridge implements LoggingFeature {
+
+    private static final boolean USE_JUL = true;
+    private static final Dependency JUL_TO_SLF4J = Dependency.builder()
+            .groupId("org.slf4j")
+            .artifactId("jul-to-slf4j")
+            .version("2.0.9")
```

Comment:
the version should be defined in https://github.com/micronaut-projects/micronaut-starter/blob/4.1.x/gradle/templates.versions.toml

Reply:
see: https://github.com/micronaut-projects/micronaut-starter/pull/2161/commits/d3d3520a0d8402a74eee01442ed06fa701ba4435